packages/gatsby-plugin-fastify/src/plugins/functions.ts

```diff
@@ -49,6 +49,8 @@ export const handleFunctions: FastifyPluginAsync<{
             handler: async function (request, reply) {
               try {
                 reply.appendModuleHeader("Functions");
+                reply.mode = "FUNCTION";
+                reply.path = `${FUNCTIONS_PREFIX}${path}`;
```

Comment:
Should we stay consistent with our use of `path.join`, or the POSIX version, which ever is correct?

Reply:
we can switch it to POSIX - just `path.join` does `\\api\\test` (on Windows).